### DISCUSSION TITLE
How to solve the server will not issue certificates for the identifier?

### DISCUSSION THREAD

**Discussion by FSeidinger-XI** [smallstep/certificates#2086] [Q&A]
I've setup small step ca for an internal network with four subnets. Let's call them a.internal, b.internal, c.internal and d.internal. When I try to get a certificate either via CLI or over acme, the step ca tells me:

The server will not issue certificates for the identifier

What do I do wrong?

**Comment by hslatman** [comment]
Hi @FSeidinger-XI,

That error type can be returned during ACME challenge validation and policy evaluation. Did you configure an authority issuance policy? What ACME client are you using? Do you have logs from the CA side?
